corebehrt/configs/performance_tests/prepare_held_out_good.yaml

```
@@ -0,0 +1,27 @@
+paths:
+## INPUTS
+  features: ./outputs/features
+  tokenized: ./outputs/tokenized
+  cohort: ./outputs/cohort/held_out/ # path to cohort directory
+  outcomes: ./outputs/outcomes
+  outcome: TEST_OUTCOME.csv
+
+## OUTPUTS
+  prepared_data: ./outputs/testing/held_out/processed_data/good/ # Save model/outputs to this folder
+  #runs: ./outputs/pretraining # Use for generating a new model folder
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)